Lazy/Lazy/Lazy/MultithreadedLazy.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Lazy;
+
+/// <summary>
+/// Multithreaded implementation of Lazy
+/// </summary>
+/// <typeparam name="T">Element type</typeparam>
+public class MultithreadedLazy<T> : Lazy<T>
+{
+    /// <summary>
+    /// Boolean variable for detecting the first call
+    /// </summary>
+    private bool isAlreadyCounted;
+
+    /// <summary>
+    /// Lock
+    /// </summary>
+    private readonly Object lockObject = new();
+
+    /// <inheritdoc/>
+    public MultithreadedLazy(Func<T> func) : base(func) { }
+
+    /// <inheritdoc/>
+    public override T Get()
+    {
+        ///if the value has already been calculated, there should be no locks
```

Comment:
Тут обычный комментарий, не XMLDoc, так что
```suggestion
        // if the value has already been calculated, there should be no locks
```